packs/data/other-effects.db/effect-2-circumstance-penalty-to-attack-rolls.json

```
@@ -0,0 +1,48 @@
+{
+    "_id": "5gX3PZztMecmWxX9",
+    "img": "systems/pf2e/icons/effects/critical-effect.webp",
+    "name": "Effect: -2 circumstance penalty to attack rolls",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #10]{Critical Fumble Deck #10}, @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #15]{Critical Fumble Deck #15}, @UUID[Compendium.pf2e.criticaldeck.Critical Hit Deck #29]{Critical Hit Deck #29}</strong></p>\n<p>-2 circumstance penalty to attack rolls</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Critical Effect",
```

```suggestion
                "label": "PF2E.SpecificRule.CriticalDeck.Failure",
```

Add "Critical Fumble Card" and "Critical Hit Card" to re-en.json in that path to localize all these.

---

Sorry, forgot I was going to do that :)

I went with `PF2E.SpecificRule.CriticalDeck.Effect.Label` as the effects are not all specific to Critical Hits vs Critical Fumbles, several effects are reused across both decks (namely a lot of the speed penalty effects) so I kept it more generic.